bundle/compliance/cis_aws/rules/cis_3_8/test.rego

```diff
@@ -0,0 +1,34 @@
+package compliance.cis_aws.rules.cis_3_8
+
+import data.cis_aws.test_data
+import data.compliance.cis_aws.data_adapter
+import data.compliance.policy.aws_kms.ensure_symmetric_key_rotation_enabled as audit
+import data.lib.test
+
+finding = audit.finding
+
+test_violation {
+	eval_fail with input as rule_input(false)
+}
+
+test_pass {
+	eval_pass with input as rule_input(true)
+}
+
```

Comment:
should we add test cases with the asymmetric keys too? what should be the result in that case?

Reply:
don't think we should send evaluation of asymmetric keys as it would be confusing for the user to have 2 types of failed findings with different reasons. also as mentioned - the title is quite specific about ensuring *rotation* which is only applicable to symmetric keys.